migration/migration/015-tracing-compression.sql

```diff
@@ -1,67 +1,339 @@
+-- Due to incompatible change of trace_id type we need to drop tables and delete data
+DROP TABLE _ps_trace.link;
```

Comment:
probably need `cascade` on these drops

Reply:
there are views referencing these tables. idempotent scripts will add the views back